src/sentry/models/sentryapp.py

```diff
@@ -135,6 +144,16 @@ def is_unpublished(self):
     def is_internal(self):
         return self.status == SentryAppStatus.INTERNAL
 
+    @property
+    def datadog_name(self):
+        if self.is_published:
+            slug = self.slug
+        if self.is_internal:
+            slug = "internal"
+        if self.is_unpublished:
+            slug = "unpublished"
```

Comment:
@ceorourke Just so you know, this sort of programming pattern of `if` statements is generally frowned upon. Better to just return the value instead of setting a variable and returning it later. My suggestion:
```
        if self.is_internal:
            return "internal"
        if self.is_unpublished:
            return "unpublished"
        return self.slug
```